docs/setup-cluster/deploy-cluster/slurm/singularity.rst

```diff
@@ -230,6 +230,16 @@ If you only want the sharable .sqsh file without the overhead of container creat
 
    manage-enroot-cache -s /shared/enroot --nocreate determinedai/environments:cuda-10.2-base-gpu-mpi-0.19.4
 
+Credentials for image download may optionally be configured by following the `enroot documentation
+<https://github.com/NVIDIA/enroot/blob/master/doc/cmd/import.md>`__. Specify the user name to use
+with the ``--username`` option:
+
+.. code:: bash
+
+   manage-enroot-cache -s /shared/enroot --username <username-here> --cuda --cpu
+
+``--username`` is positional -- if used it should appear before any image reference.
+
```

Comment:
To optionally configure credentials for image downloads, follow the `enroot documentation
<https://github.com/NVIDIA/enroot/blob/master/doc/cmd/import.md>`__. Specify the username with the ``--username`` option: